api/src/main/java/io/opentelemetry/context/propagation/BinaryFormat.java

```diff
@@ -1,49 +0,0 @@
-/*
- * Copyright 2019, OpenTelemetry Authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package io.opentelemetry.context.propagation;
-
-import javax.annotation.concurrent.ThreadSafe;
-
-/**
- * Formatter to serializing and deserializing a value with into a binary format.
- *
- * @since 0.1.0
- */
-@ThreadSafe
-public interface BinaryFormat<V> {
```

Comment:
We do need this for OpenCensus backwards compatibility. Maybe needs to be re-written a bit.